stack/stack_gfsv16.2.yaml

```diff
@@ -0,0 +1,359 @@
+cmake:
+  build: NO
+  bootstrap: NO
+  version: 3.20.1
+
+jpeg:
+  build: YES
+  shared: NO
+  version: 9.1.0
+
+zlib:
+  build: YES
+  shared: NO
+  version: 1.2.11
+
+libpng:
+  build: YES
+  shared: NO
+  version: 1.6.37
+
+szip:
+  build: YES
+  shared: NO
+  version: 2.1.1
+
+jasper:
+  build: YES
+  shared: NO
+  version: 2.0.25
+
+udunits:
+  build: YES
+  shared: NO
+  version: 2.2.28
+
+hdf5:
+  build: YES
+  version: 1.10.6
+  shared: NO
+  enable_szip: NO
+  enable_zlib: YES
+
+pnetcdf:
+  build: NO
+  version: 1.12.1
+  shared: NO
+
+netcdf:
+  build: YES
+  shared: NO
+  enable_pnetcdf: NO
+  version_c: 4.7.4
+  version_f: 4.5.3
+  version_cxx: 4.3.1
+
+nccmp:
+  build: YES
+  version: 1.8.9.0
+
+nco:
+  build: YES
+  version: 4.7.9
+
+cdo:
+  build: YES
+  version: 1.9.8
+
+pio:
+  build: YES
+  version: 2.5.3
+  enable_pnetcdf: NO
+  enable_gptl: NO
+
+esmf:
+  build: YES
+  version: 8_0_1
```

Comment:
I load `esmf_ver=8.0.1` on WCOSS2...is this the same?

Reply:
Yes